OxfordBodleian/Juel-Jensen/BDLaethd17.xml

```diff
@@ -33,87 +37,309 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                and BGV-003.</note>
+                            <!--refer to ids when they exist-->
+                        </msItem>
+
+
+
+
                     </msContents>
-                    
+
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>Some folia have holes, presumably from parchmenting: <locus
+                                            target="#26 #37 #39 #53 #70 #85 #86"/>
+                                    </p>
+                                    <p>Tears carefully amended on <locus
+                                            target="#31 #34 #85 #88 #94 #100"/>
+                                    </p>
                                 </support>
                                 <extent>
                                     <measure unit="leaf">139</measure>
-                                    <measure unit="leaf" type="blank"></measure>
-                                    <!--<locus target=""/>-->
+                                    <measure unit="leaf" type="blank"/>
+                                    <locus from="1r" to="3v"/>
+                                    <locus target="#139rb"/>
                                     <measure type="weight" unit="g">1963</measure>
                                     <dimensions type="outer" unit="mm">
                                         <height>237</height>
                                         <width>201</width>
-                                        <depth>43</depth>
+                                        <depth>73</depth>
                                     </dimensions>
                                     <dimensions type="leaf" unit="mm" xml:id="leafdim">
                                         <height>237</height>
                                         <width>199</width>
                                     </dimensions>
-                                    <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#52r"/>.</note>
+                                    <note corresp="#leafdim">Data on leaves dimensions taken from
+                                            <locus target="#52r"/>.</note>
                                 </extent>
-                                
+
+                                <foliation>Foliation in pencil in the upper right
+                                    corner.</foliation>
+
                                 <collation>
+                                    <signatures> Quire marks are written in the upper inner corner
+                                        of the recto of the first leaf of each quire with black ink.
+                                        Quire 12 (<locus from="90r" to="97v"/>) has no marking. The
```

Comment:
you could refer to the quires with `<ref target="q...">`

Reply:
nö :)